app/repositories/system.py

```diff
@@ -101,3 +103,25 @@ async def _handle_gather_hardware_info():
 async def register_hardware_info_gatherer():
     loop = asyncio.get_event_loop()
     loop.create_task(_handle_gather_hardware_info())
+
+
+async def shutdown(reboot: bool):
```

Comment:
This should be in [system_impl](https://github.com/fusion44/blitz_api/blob/main/app/repositories/system_impl/raspiblitz.py) and then imported. Other platforms might want to implement this differently.